graphql.schema.json

```diff
@@ -12,6 +12,7 @@
         "kind": "ENUM",
         "name": "ActiveStatus",
         "description": null,
+        "isOneOf": null,
```

Comment:
opened https://github.com/greenriver/hmis-frontend/pull/872 for this unrelated change

Reply:
Thanks! 👍